Patch by Robert Scheck <robert@fedoraproject.org> for gkrelltop >= 2.2.13, which
adds the missing $RPM_OPT_FLAGS to the Makefile for proper building results.

--- gkrelltop-2.2.13.orig/Makefile		2009-07-24 07:09:39.000000000 +0200
+++ gkrelltop-2.2.13.orig/Makefile.optflags	2009-07-25 01:01:50.000000000 +0200
@@ -43,14 +43,14 @@
 
 ifeq ($(GKRELL1FLAG),0)
 # Parameters for gkrellm version 2.*
-CFLAGS2 = -g -D$(OSFLAG) -DGKRELLM2 -fPIC -Wall `pkg-config gtk+-2.0 --cflags`
+CFLAGS2 = -g -D$(OSFLAG) -DGKRELLM2 -fPIC -Wall `pkg-config gtk+-2.0 --cflags` $(RPM_OPT_FLAGS)
 LIBS =  
 CC=/usr/bin/gcc $(CFLAGS2)
 
 else 
 # Parameters for gkrellm version 1.*
 CC=/usr/bin/cc
-CFLAGS = -D$(OSFLAG) -Wall -fPIC `gtk-config --cflags` `imlib-config --cflags-gdk` 
+CFLAGS = -D$(OSFLAG) -Wall -fPIC `gtk-config --cflags` `imlib-config --cflags-gdk` $(RPM_OPT_FLAGS)
 LIBS=
 
 endif
@@ -65,12 +65,12 @@
 endif
 
 ifeq ($(WANT_GLIB12),yes)
-CFLAGSD = -I/usr/X11R6/include -D$(OSFLAG) -fPIC -Wall `glib-config --cflags`
+CFLAGSD = -I/usr/X11R6/include -D$(OSFLAG) -fPIC -Wall `glib-config --cflags` $(RPM_OPT_FLAGS)
 LIBSD = `glib-config --libs`
 LIBSD = 
 CONFIGURE_ARGS += --with-glib12
 else
-CFLAGSD = -D$(OSFLAG) -fPIC -Wall `pkg-config glib-2.0 --cflags`
+CFLAGSD = -D$(OSFLAG) -fPIC -Wall `pkg-config glib-2.0 --cflags` $(RPM_OPT_FLAGS)
 LIBSD = `pkg-config glib-2.0 --libs`
 LIBSD = 
 endif
